packages/ogmios/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './ogmiosTxSubmitProvider';
+export { createConnectionObject } from '@cardano-ogmios/client';
```

Comment:
I think this is fine. Re-exports to avoid are for `core` package types, as we want them to be stable and they tend to benefit from some changes compared to ogmios types (opaque strings with validation etc.).